actions/class.Items.php

```diff
@@ -359,6 +359,16 @@ public function authoring()
                         LockManager::getImplementation()
                             ->setLock($item, $this->getSession()->getUser()->getIdentifier());
 
+                        // TODO: better implement the following parameters
+                        // --- start
+                        if ($this->getRequestParameter('translation') !== null) {
+                            $authoringUrl .= '&translation=' . $this->getRequestParameter('translation');
```

Comment:
sounds good to me, thanks! 

If you want, you can implement sprintf for safer concatenation. Something like

```
$authoringUrl = sprintf('%s&translation=%s', $authoringUrl, $this->getRequestParameter('translation'));
```

Reply:
Good suggestion, thanks!